supabase/tests/69_test_channel_overrides_require_membership.sql

```
@@ -0,0 +1,201 @@
+      is_allowed
+    )
+    VALUES (
+      public.rbac_principal_user(),
+      tests.get_supabase_uid('channel_override_outsider'),
+      6900401,
+      public.rbac_perm_channel_promote_bundle(),
+      true
+    )
+  $$,
+  '42501',
+  'new row violates row-level security policy for table "channel_permission_overrides"',
```

SQLFluff (4.3.0)

[error] Line is too long (88 > 80).

(LT05)